### PR TITLE
Change TurnkeyProject endpoints + add position

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: features/api/turnkey_projects.feature [hunk 1]
```diff
@@ -28,30 +28,34 @@ Feature:
       "category":"Nature et Environnement",
       "title":"Stop mégots !",
       "slug":"stop-megots",
-      "subtitle":"Campagnes de sensibilisation et de revalorisation des mégots jetés"
+      "subtitle":"Campagnes de sensibilisation et de revalorisation des mégots jetés",
+      "position": 1
```
Comment (ottaviano) [inline features/api/turnkey_projects.feature]: why do we need to have `position` field in the response, the ordered list (without position field) is not enough?

File: src/Controller/Api/TurnkeyProjectController.php [hunk 2]
```diff
@@ -13,59 +13,62 @@
...
-            [],
-            true
+            ['total' => $repository->countApproved()],
+            JsonResponse::HTTP_OK
         );
     }
 
     /**
-     * @Route("/turnkey-projects/count", name="api_count_approved_turnkey_projects")
+     * @Route("/is-pinned", name="api_pinned_turnkey_project")
```
Comment (ottaviano) [inline src/Controller/Api/TurnkeyProjectController.php]: 1) what about just `/pinned`?
2) `api_pinned_turnkey_project` => `api_pinned_turnkey_projectS` 
3) is the list of pinned projects ordered?

Comment (illusionOfParadise) [reply src/Controller/Api/TurnkeyProjectController.php]: 1. Why not `is-pinned`?
2. No, there is only one pinned turnkey project, it's not possible to have more than one.
3. Same that 2.

File: features/api/turnkey_projects.feature (hunk 1 above)
Comment (illusionOfParadise) [reply features/api/turnkey_projects.feature]: Yes, there is no need of position. I removed it.

File: src/Controller/Api/TurnkeyProjectController.php [hunk 3]
```diff
@@ -13,59 +13,62 @@
...
      * @Method("GET")
      */
-    public function countApprovedTurnkeyProjectAction(TurnkeyProjectRepository $repository): Response
+    public function getPinnedTurnkeyProjectAction(TurnkeyProjectRepository $repository, Serializer $serializer): Response
     {
+        $turnkeyProject = $repository->findPinned();
```
Comment (thomasbisignani) [inline src/Controller/Api/TurnkeyProjectController.php]: This variable is useless, it is used once

File: src/Controller/Api/TurnkeyProjectController.php (hunk 2 above)
Comment (thomasbisignani) [reply src/Controller/Api/TurnkeyProjectController.php]: I agree with the naming `/pinned`, it is more in accordance with the action name, and `is-pinned` make us think boolean logic.

**Pull request merged**